src/components/Dashboard/donation_requests_chart.js

```diff
@@ -65,37 +66,52 @@ export default class DonationRequestsChart extends Component {
             borderDash: [],
             borderDashOffset: 0.0,
             data: this.props.data,
+            is_admin: null,
           },
         ],
       },
     };
-   
   }
 
   componentDidUpdate(prevProps) {
     if (prevProps.data !== this.props.data) {
       console.log(this.props.data);
-      let { chartData } = this.state;
+      let  chartData  = {...this.state.chartData};
       chartData.datasets[0].data = this.props.data;
       this.setState({ chartData });
     }
   }
+  
+  checkIsAdmin = () => {
+    let decodedToken = jwt_decode(UserUtils.getAccessToken());
```

Comment:
we shouldn't have to decode the token for this, we can store user permissions in local storage, what happens if the user has privileges to see requests that are pending and but is not an admin?